.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,73 @@
+name: Release Artifacts
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+**'
+    branches:
+      - 'release/**'
+      - 'develop'
```

Comment:
Add your branch here once, to see if this runs fine

Reply:
Will it work. Its meant run on tag push right? Did you mean to test `pull_request_target` on `pr-image.yml ` test?

Reply:
Yes it will work